app/code/Magento/Quote/Test/Unit/Model/Quote/Item/ProcessorTest.php

```diff
@@ -447,4 +447,30 @@ public function testPrepareWithResetCountAndNotStickAndSameItemId()
 
         $this->processor->prepare($this->itemMock, $this->objectMock, $this->productMock);
     }
+
+    public function testPrepareWithCustomPrice()
```

Comment:
Please provide a proper method description and change the method name with its actual case (I'd mention 0 as custom price).
Also, could you please add the return type.
```suggestion
    public function testPrepareWithCustomPrice(): void
```

Reply:
Hi @Bal2018, please add the method description here.